src/components/info/UserProfile.js

```diff
@@ -27,13 +36,46 @@ export default function UserProfile() {
     removeAndNavigateSubLink(pathname + hash, "/profile")
   );
 
+  const sendLogout = async () => {
```

Comment:
we have same code in `src/components/navigation/NavigationLine.js`

Can we re-use it somehow and do not duplicate?